src/target/imxrt.c

```diff
@@ -31,7 +31,6 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <string.h>
```

Comment:
We know why you removed this from Discord, but if memory serves this then causes include errors in VSCode on some platforms. Might we suggest moving the inclusion below general.h but keeping it present?